asynq/tools.py

```diff
@@ -305,3 +307,39 @@ def resume(self):
 
     def pause(self):
         self.total_time += utime() - self._last_start_time
+
+
+class AsyncEventHook(EventHook):
+    """EventHook that supports async handlers.
+
+    When the event triggers, all the handlers will be invoked asynchronously using async_call.
```

Comment:
I feel like async_call is an implementation detail that doesn't need to be mentioned here. What matters for users of the class is that both async and non-async handlers are supported.